materialize-rockset/.snapshots/TestRocksetDriverSpec-TestEndpointSpecSchema

```diff
@@ -2,16 +2,30 @@
   "$schema": "http://json-schema.org/draft/2020-12/schema",
   "$id": "https://github.com/estuary/connectors/materialize-rockset/config",
   "properties": {
+    "region_base_url": {
```

Comment:
Back when we reviewed the first version of the connector with them, we wrote up #161. As mentioned in the issue, their recommendation was to let the client determine the region automatically. I'm guessing you found some problems with that? I'm wondering (hoping) whether this is another case where we should report it and see if they can fix it in their client library. It would be unfortunate for us to have to keep updating these urls every time Rockset expands to a new region or cloud provider.

Reply:
I've confirmed that you do need to set a base URL now. Previously the client would just use the default, which was a US west cluster, but now that there are more clusters it cannot automatically determine which one to use and will error if a base url isn't set. This is the intended behavior of the client & API.

Making this an enum list is a little overkill, but I wanted to make it as smooth as possible to get working in the short term since I could see there being a lot of friction in getting the URL correct without a list to pick from. I think we'll want to make it free-form entry in the future if more URLs added and we have some traction with the connector.